test/search/hex_client_test/release_test.exs

```diff
@@ -0,0 +1,44 @@
+defmodule Search.HexClientTest.ReleaseTest do
+  use ExUnit.Case, async: true
+
+  alias Search.HexClient.Release
+
+  describe "latest/1" do
+    test "when given no prerelease entries, returns the release with largest version" do
+      releases = [
+        %Release{package_name: "lesser", version: Version.parse!("1.2.3")},
+        %Release{package_name: "greater", version: Version.parse!("2.1.3")},
+        %Release{package_name: "middle", version: Version.parse!("1.4.3")}
+      ]
+
+      assert %Release{package_name: "greater"} =
+               Release.latest(releases)
```

Comment:
nitpick: this doesn't matter so I'm ok with keeping this as is but in practice we'd have the same package name but different version so I'd consider changing this and similar tests to be slightly more realistic:

```suggestion
      releases = [
        %Release{version: Version.parse!("1.2.3")},
        %Release{version: Version.parse!("2.1.3")},
        %Release{version: Version.parse!("1.4.3")}
      ]

      assert Release.latest(releases).version == Version.parse!("2.1.3")
```